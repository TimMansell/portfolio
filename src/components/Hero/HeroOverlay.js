import React from 'react';

import useScrollBlur from 'hooks/useScrollBlur';

import styles from './Hero.module.scss';

export const HeroOverlay = () => {
  const scrollStyles = useScrollBlur(0, 10);

  return <div className={styles.overlay} style={scrollStyles}></div>;
};

export default HeroOverlay;
