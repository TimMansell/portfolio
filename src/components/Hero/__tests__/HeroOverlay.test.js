import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import HeroOverlay from '../HeroOverlay';

describe('HeroOverlay', () => {
  it('should render my component', () => {
    render(<HeroOverlay />);
  });

  it('should match snapshot', () => {
    const { asFragment } = render(<HeroOverlay />);

    expect(asFragment()).toMatchSnapshot();
  });
});
