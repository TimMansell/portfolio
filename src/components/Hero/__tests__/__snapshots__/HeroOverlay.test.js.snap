// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroOverlay should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="overlay"
  />
</DocumentFragment>
`;
